src/test/java/org/cactoos/io/ZipTest.java

```diff
@@ -67,7 +67,7 @@ public void zip() throws Exception {
                 "Can't list files in a directory represented by a path",
                 cnt,
                 // @checkstyle MagicNumber (1 line)
-                Matchers.equalTo(4)
+                Matchers.equalTo(1)
```

Comment:
@fanifieiev please convert this test to `Assertion` and OO matchers instead of static method. Don't forget about changing the assertion message to an assertive sentence ("Must..." instead of "Can't...").